olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/CompatibleVersionDependency.h

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright (C) 2022 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#pragma once
+
+#include <olp/dataservice/read/DataServiceReadApi.h>
+#include <string>
+#include <utility>
+#include <vector>
+
+namespace olp {
+namespace dataservice {
+namespace read {
+
+/// Represents dependent catalogs.
+class DATASERVICE_READ_API CompatibleVersionDependency {
```

Comment:
Looking at the schema, I see `Dependencies` holding a list of `CatalogVersion`. We should name the classes same as the schema defines them.
Should be similar to `olp::dataservice::read::model::VersionInfos` and `olp::dataservice::read::model::VersionInfo`.
Please also put them under the `model` folder and namespace.